deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -33,44 +33,21 @@ collectors:
   #  Cluster is a K8s deployment EDOT collector focused on gathering telemetry
   #  at the cluster level (Kubernetes Events and cluster metrics).
   cluster:
-    # Configure the pods resources to control CPU and memory usage.
-    # resources:
-    #   limits:
-    #     cpu: 100m
-    #     memory: 500Mi
-    #   requests:
-    #     cpu: 100m
-    #     memory: 500Mi
     env:
       - name: ELASTIC_AGENT_OTEL
         value: '"true"'
-      - name: ELASTIC_ENDPOINT
-        valueFrom:
-          secretKeyRef:
-            name: elastic-secret-otel
-            key: elastic_endpoint
-      - name: ELASTIC_API_KEY
-        valueFrom:
-          secretKeyRef:
-            name: elastic-secret-otel
-            key: elastic_api_key
     config:
       exporters:
         # [Debug exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/debugexporter/README.md)
         debug:
           verbosity: basic # Options: basic, detailed. Choose verbosity level for debug logs.
           # [Elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter/README.md)
-        elasticsearch/otel:
-          endpoints: # List of Elasticsearch endpoints.
-            - ${env:ELASTIC_ENDPOINT}
-          api_key: ${env:ELASTIC_API_KEY} # API key for Elasticsearch authentication.
-          logs_dynamic_index:
-            enabled: true
-          # Enable in order to skip the SSL certificate Check
-          # tls:
-          #   insecure_skip_verify: true
-          mapping:
-            mode: otel
+        otlp/gateway:
+          # !!! Elastic APM https endpoint WITHOUT the "https://" prefix
+          endpoint: "http://opentelemetry-kube-stack-gateway-collector:4317"
+          compression: none
```

Comment:
Why are we disabling compression here?

Reply:
Ah, nice catch. I think this was a leftover from a configuration that was connecting to the `apm-server`. Now the otlp connections are within the cluster (otlp receiver/exporters), we can enable it again https://github.com/elastic/elastic-agent/pull/6444/commits/d538b4b4f057eec188083ce628a8974875fd990f